app/assets/javascripts/admin/resources/services/orders.js.coffee

```diff
@@ -9,7 +9,7 @@ angular.module("admin.resources").factory 'Orders', ($q, OrderResource, RequestM
       request = OrderResource.index params, (data) =>
         @load(data)
         (callback || angular.noop)(data)
-      RequestMonitor.load(request.$promise)
```

Comment:
Is it really okay to remove this line? It doesn't seem necessary to remove, and many pages including the bulk order management use `RequestMonitor`.

Reply:
That call was redundant. The only call of this method is in the line items controller and it uses the RequestMonitor already:
https://github.com/openfoodfoundation/openfoodnetwork/blob/d129665d439671d82eeba89f7cb8c80e4eb0920a/app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee#L32